internal/res/default.toml

```diff
@@ -155,6 +155,10 @@ freeze_at = 0
 # The percentage to show instances at on preview. Set below 0 to disable.
 show_at = 5
 
+# The name of the projector window to consider as the wall screen
```

Comment:
I think this comment should be adjusted to state that `wall_window` does not have to be the full window name, just a part of it.

Nitpicks: English should be capitalized and each line should end with a period.